bazel-mode.el

```diff
@@ -21,6 +21,37 @@
 ;;
 ;;; Code:
 
+(defgroup bazel-mode nil
+  "Major mode for Bazel BUILD files."
+  :link '(url-link "https://github.com/bazelbuild/emacs-bazel-mode")
+  :group 'languages)
+
+(defcustom bazel-mode--buildifier-cmd "buildifier"
```

Comment:
This should probably not internal if the user is supposed to customize it. See the second item at https://www.gnu.org/software/emacs/manual/html_node/elisp/Coding-Conventions.html for symbol naming conventions.

Reply:
This previously did not have a package name prefix, but the previous comment on it was "Please start all names with the name of the package, as Emacs Lisp doesn't have namespaces. See https://www.gnu.org/software/emacs/manual/html_node/elisp/Coding-Conventions.html."

To clarify for here and for the interactive `buildidifer` command, both are part of the user-visible interface for this package, so I'm thinking both should not have the prefix. Is that right?

Reply:
These are two different things:

- All global symbols (functions, dynamic variables, etc.) need to be prefixed with the package name because all such symbols from all packages share a single global namespace. Local variables don't need the prefix because they are always bound temporarily.

- Internal symbols should be marked as such by having a double hyphen (`--`) after their name. Public symbols that are part of the user interface should not have a double hyphen.

These two concepts (global vs. local and public vs. internal). Typically you have a lot of global symbols, which can be either public or internal.

Reply:
Thanks for the clarification.